a3p-integration/package.json

```diff
@@ -4,7 +4,10 @@
         "fromTag": "latest"
     },
     "scripts": {
-        "build": "../scripts/generate-a3p-submission.sh && yarn synthetic-chain build",
+        "build": "yarn run build:sdk && yarn run build:submission && yarn run build:synthetic-chain",
+        "build:sdk": "make -C ../packages/deployment docker-build-sdk",
```

Comment:
This is going to be painful since that takes several minutes. But I agree it's the right thing to do to avoid surprises. Anyone who wants to save that time can run more targeted commands.

I've spent a little time trying to speed it up but it's pretty hairy
- https://github.com/Agoric/agoric-sdk/issues/8848

Reply:
I believe now it only rebuilds if the source has changed

Reply:
Oh I suppose the `GIT_COMMIT` and `GIT_REVISION` are gonna be a problem busting the cache

Reply:
yeah. and the way they're read it can't even cache when they haven't changed https://github.com/Agoric/agoric-sdk/pull/8850